analysis/src/SharedTypes.ml

```diff
@@ -425,6 +425,12 @@ module Completable = struct
     | CPObj of contextPath * string
     | CPPipe of contextPath * string
 
+  type typedContext = NamedArg of string
```

Comment:
You can add a comment: the argument name of a function type.

Reply:
I made it a record instead and added comments.